pkg/options/options.go

```diff
@@ -88,6 +90,7 @@ func (o *Options) AddFlags() {
 	o.flags.Var(&o.Namespaces, "namespace", fmt.Sprintf("Comma-separated list of namespaces to be enabled. Defaults to %q", &DefaultNamespaces))
 	o.flags.Var(&o.MetricWhitelist, "metric-whitelist", "Comma-separated list of metrics to be exposed. This list comprises of exact metric names and/or regex patterns. The whitelist and blacklist are mutually exclusive.")
 	o.flags.Var(&o.MetricBlacklist, "metric-blacklist", "Comma-separated list of metrics not to be enabled. This list comprises of exact metric names and/or regex patterns. The whitelist and blacklist are mutually exclusive.")
+	o.flags.Var(&o.AnnotationLabelWhitelist, "annotation-label-whitelist", "Comma-separated list of annotation labels to be exposed. This list comprises of exact annotation label names and/or regex patterns.")
```

Comment:
This needs a warning for users enabling this:

> Warning: annotations might include sensitive information and produce unbound high cardinality.

cc @brancz @tariq1890 sgty ^
```suggestion
	o.flags.Var(&o.AnnotationLabelWhitelist, "annotation-label-whitelist", "Comma-separated list of annotation labels to be exposed. This list comprises of exact annotation label names and/or regex patterns. Warning: annotations might include sensitive information and produce unbound high cardinality.")
```

Reply:
Hi @LiliC, I will add the metrics which are currently not documented and failing the tests.

Reply:
@lilic ready to review again :)